internal/safety/safety_bench_test.go

```diff
@@ -0,0 +1,140 @@
+package safety
+
+import (
+	"testing"
+
+	"github.com/vdaas/vald/internal/errors"
+)
+
+func BenchmarkRecoverFunc(b *testing.B) {
+	type args struct {
+		fn func() error
+	}
+	type test struct {
+		name string
+		args args
+	}
+	tests := []test{
+		{
+			name: "fn return error",
```

Comment:
is it ok to name it `fn return error`?

Reply:
🤔 
for example, how about `fn function that returns an error`?

Reply:
`fn return error func`

Reply:
@hlts2 what do you think about vankichi suggestion?

Reply:
@kevindiu 
sorry, it looks good to me 👍 
cc: @vankichi